lib/utilities/pristine.js

```diff
@@ -85,7 +86,9 @@ function installPristineApp(appName, options) {
 
   chdir(temp.pristinePath);
 
-  var promise = applyCommand('new', appName, extraOptions)
+  var args = extraOptions.concat(['-sb', '-sn', runCommandOptions]);
```

Comment:
I saw that npm and bower are installing later anyways:
https://github.com/tomdale/ember-cli-addon-tests/pull/24/files#diff-32914fa89b187f0dba3ec3832f5ad335R109
https://github.com/tomdale/ember-cli-addon-tests/pull/24/files#diff-32914fa89b187f0dba3ec3832f5ad335R123

I figured we could save some cycles by skipping it here.